feat(jobs): troubleshooting missing metrics (#4130)
scaleway/docs-content: serverless/jobs/troubleshooting/missing-metrics.mdx

---
meta:
  title: Missing metrics for my Job Run
  description: Troubleshoot missing metrics for Scaleway Serverless Jobs.
content:
  h1: Missing metrics for my Job Run
  paragraph: Troubleshoot missing metrics for Scaleway Serverless Jobs.
tags: serverless jobs troubleshooting issue missing metrics
dates:
  validation: 2024-12-19
  posted: 2024-12-19
categories:
  - serverless
---

## Problem

I can't see metrics partially or totally for my Serverless Job.

## Cause

Metrics for Serverless Jobs technically limited for very short Jobs under `15` seconds.

## Possible solutions

> - If your Job takes more than `15` seconds and metrics are still missing, it can be due to a [scheduled maintenance](https://status.scaleway.com/). If this is not the case, you can open a support ticket to report the issue. 

Reviewer: ```suggestion
- If your job takes more than `15` seconds and metrics are still missing, it can be due to a [scheduled maintenance](https://status.scaleway.com/). If this is not the case, you can open a support ticket to report the issue. 
```

---
meta:
  title: Missing metrics for my Job Run
  description: Troubleshoot missing metrics for Scaleway Serverless Jobs.
content:
  h1: Missing metrics for my Job Run
  paragraph: Troubleshoot missing metrics for Scaleway Serverless Jobs.
tags: serverless jobs troubleshooting issue missing metrics
dates:
  validation: 2024-12-19
  posted: 2024-12-19
categories:
  - serverless
---

## Problem

I can't see metrics partially or totally for my Serverless Job.

## Cause

Metrics for Serverless Jobs technically limited for very short Jobs under `15` seconds.

## Possible solutions

> - If your Job takes more than around `15` seconds and metrics are still missing, it can be due to a [scheduled maintenance](https://status.scaleway.com/). If that's not the case please open a support ticket.

Reviewer 2: ```suggestion
- If your Job takes more than `15` seconds and metrics are still missing, it can be due to a [scheduled maintenance](https://status.scaleway.com/). If this is not the case, you can open a support ticket to report the issue. 
```

> - For very short Jobs, if you need the metrics you can add a `sleep` in your workflow.

Reviewer 2: ```suggestion
- For very short Jobs, you can add `sleep` to your workflow if you need the metrics.
```

> ### Note
> 
> Gradually we will improve metrics for short Jobs.

Reviewer 2: ```suggestion
```
